charts/opentelemetry-operator/templates/_helpers.tpl

```diff
@@ -150,3 +150,9 @@ The image to use for opentelemetry-operator.
 {{- define "opentelemetry-operator.image" -}}
 {{- printf "%s:%s" .Values.manager.image.repository (default .Chart.AppVersion .Values.manager.image.tag) }}
 {{- end }}
+
+{{- define "opentelemetry-operator.featureGatesMap" -}}
```

Comment:
If we are deprecating `.Values.manager.featureGates` then this helper needs to be able to handle `.Values.manager.featureGatesMap` containing multiple feature gates.